gallery/README.md

```diff
@@ -0,0 +1,30 @@
+# Flutter  Gallery
+
+Flutter Gallery is a resource to help developers evaluate and use Flutter.
+It is a collection of material design widgets, behaviors, and vignettes
+implemented with Flutter. We often get asked how one can see Flutter in
+action, and this gallery helps them see what Flutter provides and how it
```

Comment:
"Them" doesn't have an antecedent in this sentence. Consider something like "...this gallery demonstrates what Flutter provides..." instead.

Reply:
good call, updated